CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 8.0.0
+
+-   Added: stylelint semantic version policy
```

Comment:
Shall we categorise this as a "change" and reference the docs.

e.g.

"Changed: stylelint [semantic version policy](README.md#semantic-versioning-policy). We now recommend using the tilde (`~`) in `package.json` e.g. `"stylelint": "~7.2.0"` to guarantee the results of your builds."

We always had an informal and undocumented policy, so I think "Changed" is dandy. Using "Changed" usually means action is required too.